<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>form_round</class>
 <widget class="QDialog" name="form_round">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>309</width>
    <height>564</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Раунд</string>
  </property>
  <widget class="QListWidget" name="All_Question">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>241</width>
     <height>341</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::EditTrigger::DoubleClicked|QAbstractItemView::EditTrigger::SelectedClicked</set>
   </property>
   <property name="showDropIndicator" stdset="0">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Del_Quest">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>520</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Удалить вопрос</string>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
   <property name="autoRepeatDelay">
    <number>388</number>
   </property>
   <property name="autoRepeatInterval">
    <number>31</number>
   </property>
  </widget>
  <widget class="QPushButton" name="Add_Quest">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>520</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить вопрос</string>
   </property>
  </widget>
  <widget class="QLabel" name="Blitz_label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>130</y>
     <width>81</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Очки блиц</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="Test_Round">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>50</y>
     <width>21</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSpinBox" name="Second_Round">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>90</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::FocusPolicy::WheelFocus</enum>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::ContextMenuPolicy::DefaultContextMenu</enum>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LayoutDirection::LeftToRight</enum>
   </property>
   <property name="wrapping">
    <bool>true</bool>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::ButtonSymbols::NoButtons</enum>
   </property>
   <property name="accelerated">
    <bool>true</bool>
   </property>
   <property name="keyboardTracking">
    <bool>true</bool>
   </property>
   <property name="maximum">
    <number>300</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="stepType">
    <enum>QAbstractSpinBox::StepType::DefaultStepType</enum>
   </property>
   <property name="value">
    <number>50</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_23">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>91</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Тестовый раунд</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_24">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>81</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Время раунда</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="Blitz_Score">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>130</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::FocusPolicy::WheelFocus</enum>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::ContextMenuPolicy::DefaultContextMenu</enum>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LayoutDirection::LeftToRight</enum>
   </property>
   <property name="wrapping">
    <bool>true</bool>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::ButtonSymbols::NoButtons</enum>
   </property>
   <property name="accelerated">
    <bool>true</bool>
   </property>
   <property name="keyboardTracking">
    <bool>true</bool>
   </property>
   <property name="maximum">
    <number>300</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="stepType">
    <enum>QAbstractSpinBox::StepType::DefaultStepType</enum>
   </property>
   <property name="value">
    <number>50</number>
   </property>
  </widget>
  <widget class="QComboBox" name="Type_Round">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>91</width>
     <height>28</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>classical</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>blitz</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>70</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Тип раунда</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
